test/Microsoft.AspNet.Mvc.TagHelpers.Test/InputTagHelperTest.cs

```diff
@@ -512,12 +566,19 @@ public class InputTagHelperTest
                 { "type", "datetime" },
                 { "value", "2014-10-15T23:24:19.000-7:00" },
             };
+            var expectedPreContent = "original pre-content";
             var expectedContent = "original content";
+            var expectedPostContent = "original post-content";
             var expectedTagName = "input";
 
-            var tagHelperContext = new TagHelperContext(new Dictionary<string, object>(), uniqueId: "test");
-            var output = new TagHelperOutput(expectedTagName, expectedAttributes, expectedContent)
+            var context = new TagHelperContext(allAttributes: new Dictionary<string, object>(),
```

Comment:
don't rename variables for consistency with other files or personal preference.  this leads to extra noise in the PR and unnecessary churn.  previous name was fine and actually appears more times in this project than unqualified `context`.